.jenkins/nightly.groovy

```diff
@@ -27,12 +27,18 @@ pipeline {
                           && \
                           apt-get clean && rm -rf /var/lib/apt/lists/*
 
+                          export SPACK_CDASH_ARGS="" && \
+                          export SPACK_CDASH_ARGS="${SPACK_CDASH_ARGS} --cdash-upload-url=https://my.cdash.org/submit.php?project=Kokkos" && \
+                          export SPACK_CDASH_ARGS="${SPACK_CDASH_ARGS} --cdash-track=Nightly" && \
+                          export SPACK_CDASH_ARGS="${SPACK_CDASH_ARGS} --cdash-site=ornl-jenkins" && \
+                          export SPACK_CDASH_ARGS="${SPACK_CDASH_ARGS} --cdash-build=spack-serial" && \
```

Comment:
I meant using the environment directive in the jenkinsfile.
Example here https://github.com/kokkos/kokkos/blob/980c8a4c4a4fc9501244620771845c5ac62a72bc/.jenkins/continuous.groovy#L43-L48

Reply:
The part where w/o the build name to avoid repeating ourselves